src/generic/components/ecl-navigation/ecl-navigation-side/ecl-navigation-side.twig

```diff
@@ -0,0 +1,86 @@
+{#
+  Parameters:
+    - "menu_label" (integer): max number of items to display (0 = display all) (default: 5)
+    - "menu_aria_label" (string): label displayed before the list (default: '')
+    - "links" (array): [{
+        "href" (string): target of the link
+        "label" (string): label of the link
+        "is_active" (boolean): does the menu contains the current page? (default: false)
+        "children_links" (array): [{
+          "href" (string): target of the link
+          "label" (string): label of the link
+          "is_active" (boolean): is it the current page? (default: false)
+        }]
+      }]
+    - "extra_classes" (string): extra CSS classes to be added
+    - "extra_attributes" (array): extra attributes classes (optional, format: [{ 'name': 'name_of_the_attribute', 'value': 'value_of_the_attribute'}])
+  Blocks:
+    - "navigation": instead of providing an array of links, you can also embed the template and fill the "navigation" block directly
+#}
+
+{% set menu_label = menu_label|default('') %}
+{% set menu_aria_label = menu_aria_label|default('') %}
+
+{# Internal properties #}
+
+{% set _css_class = 'ecl-navigation-side' %}
+{% set _extra_attributes = '' %}
+
+{# Internal logic - Process properties #}
+
+{% if extra_classes is defined %}
+  {% set _css_class = _css_class ~ ' ' ~ extra_classes %}
+{% endif %}
+
+{% if extra_attributes is defined %}
+  {% for attr in extra_attributes %}
+    {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name ~ '="' ~ attr.value ~'"' %}
+  {% endfor %}
+{% endif %}
+
+{# Print the result  #}
+
+<nav class="{{ _css_class }}" aria-label="{{ menu_aria_label }}"{{ _extra_attributes|raw }}>
+  <button class="ecl-navigation-side__toggle ecl-navigation-side__hamburger ecl-navigation-side__hamburger--squeeze" aria-controls="side-menu-expandable-root" aria-expanded="false">
+    <span class="ecl-navigation-side__hamburger-box">
+      <span class="ecl-navigation-side__hamburger-inner"></span>
+    </span>
+    <span class="ecl-navigation-side__hamburger-label">{{ menu_label }}</span>
+  </button>
+  <ul class="ecl-navigation-side__root ecl-navigation-side--depth-1" id="side-menu-expandable-root" aria-hidden="true">
+    {% block navigation %}
+
+      {% for _link in links %}
+        {% set _link_class_name = 'ecl-navigation-side__link' ~ (_link.is_active ? ' ecl-navigation-side__link--active' : '') %}
+        <li class="ecl-navigation-side__item">
+          {% if _link.children_links is not defined or _link.children_links is empty %}
+            <a href="{{ _link.href }}" class="{{ _link_class_name }}">{{ _link.label }}</a>
```

Comment:
Not too problematic, but if you want to make so links respect possibly additional attributes, such as "title", maybe using the component would be beneficial, example from other part of the codebase
```
      {% include '@ec-europa/ecl-links' with {
        'extra_classes': 'ecl-breadcrumbs__link',
        'variant': ['inverted', 'standalone'],
        }|merge(segment|default({}))
      %}
```